### ISSUE TITLE
Regression: several SubScript applications now working incorrectly

### ISSUE THREAD

**Issue by AndreVanDelft** [AndreVanDelft/scala#20]
LookupFrame2, GraphicalDebugger and LifeFrame currently behave badly.
This is a regression from commit  d49faa7b0e8c6c6c916258d856e41f84658d175e [d49faa7](8 April 2014).
OperatorsSuite reports no errors.

The output from SimpleScriptDebugger in both the HEAD and the d49faa7 commit on a "sensitive" application should highlight where the problem resides.

Then the problem should be reproduced in a new test in OperatorsSuite, and thereafter resolved.

**Comment by AndreVanDelft** [comment]
Not investigated yet; I think though that this may have been introduced by replacing AAStarted and AAEnded by AAHappened graph messages; this could be sensive in case of threaded code fragments.

**Comment by anatoliykmetyuk** [comment]
I doubt it, because I remember I have debugged subscript actor-based program after AAHappened was introduced and everything worked fine. I thing something goes wrong during stepButton conversion to a script. Though exitButton apparently converts normally.

**Comment by AndreVanDelft** [comment]
Maybe it is the handling of not-normal code fragments that goes wrong; there the move from AAStarted/AAEnded to AAHappened could make a difference.

Anyway, my strategy to investigate will probably be:
- find a simple program that goes wrong
- encode this in OperatorsSuite
- run it in OperatorsSuite using the debug option
- also do so in the d49faa7 commit 
- compare the two debug outputs
- identify the first point where the behavior diverses significantly
- find the cause for that

**Comment by AndreVanDelft** [comment]
It went a bit differently; the problem appeared not to be due to AAStarted/AAEnded/AAHappened.
I ran the LookupFrame2 application with the SimpleScriptDebugger:

```
qbin/scala subscript.vm.SimpleScriptDebuggerApp  subscript.examples.LookupFrame2
```

Clicking the Exit button triggered the failure: after saying No to the "Are you sure" dialog, the Exit button would not be enabled any more. The loop was failing. 
Comparing the SimpleScriptDebugger output with that of commit [d49faa7] revealed that the Success message for a while operand was not handled any more. This was due to a bug introduced on 10 April 2014 in commit [c8f7a57]
